Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "ykbf"
+version = "0.1.0"
+authors = ["Laurence Tratt <laurie@tratt.net>"]
+edition = "2018"
```

Comment:
Probably should add a license directive.

Reply:
Agreed.